app/components/isp-jam-auth/template.hbs

```diff
@@ -14,6 +14,9 @@
       </div>
       <div class="form-group">
         <button disabled={{disableLogin}} class="btn btn-primary" onclick={{action 'authenticate'}}>{{t 'global.continueLabel'}}</button>
+          {{#if authenticating}}
+              <i class="fa fa-spinner fa-lg fa-spin"></i>
```

Comment:
Happy to offer advice/answer questions on that part, but otherwise decision to merge this one will be left to the developer.